doc/ocp_lock/lock_spec.ocp

```diff
@@ -892,151 +887,130 @@ Command Code: 0x474D_504B ("GMPK")
-| Name               | Type             | Description                            |
-+====================+==================+========================================+
-| chksum             | u32              | Checksum over other input arguments,   |
-|                    |                  | computed by the caller. Little endian. |
-+--------------------+------------------+----------------------------------------+
-| reserved           | u32              | Reserved.                              |
-+--------------------+------------------+----------------------------------------+
-| mpk_algorithm      | u32              | Indicates the size of MPKs.            |
-|                    |                  | Only one bit shall be reported:        |
-|                    |                  |                                        |
-|                    |                  | - Byte 0 bit 0: 256 bits               |
-+--------------------+------------------+----------------------------------------+
-| info_len           | u16              | Length of the info argument.           |
-+--------------------+------------------+----------------------------------------+
-| info               | u8[info_len]     | Info argument to use with HPKE unwrap. |
-+--------------------+------------------+----------------------------------------+
-| wrapped_access_key | WrappedAccessKey | KEM-wrapped access key:                |
-|                    |                  |                                        |
-|                    |                  | - access_key_algorithm                 |
-|                    |                  | - kem_handle                           |
-|                    |                  | - kem_algorithm                        |
-|                    |                  | - kem_ciphertext                       |
-|                    |                  | - encrypted_access_key                 |
-+--------------------+------------------+----------------------------------------+
++-------------------+-----------------+----------------------------------------+
+| Name              | Type            | Description                            |
++===================+=================+========================================+
+| chksum            | u32             | Checksum over other input arguments,   |
+|                   |                 | computed by the caller. Little endian. |
++-------------------+-----------------+----------------------------------------+
+| reserved          | u32             | Reserved.                              |
++-------------------+-----------------+----------------------------------------+
+| info_len          | u32             | Length of the info argument.           |
++-------------------+-----------------+----------------------------------------+
+| info              | u8[info_len]    | Info argument to use with HPKE unwrap. |
```

Comment:
Here (and elsewhere), from a programming POV, it's best if the variable-length part comes last so the structure can use a library like `zerocopy`.

Reply:
Some of these have multiple variable length items in a single struct. To make `zerocopy` happy, we plan to add max sizes for these values so they can have fixed offsets. Does that sound okay to you?